package.json

```diff
@@ -24,7 +23,7 @@
     "websocket-extensions": "^0.1.4"
   },
   "scripts": {
-    "start": "PORT=8000 react-scripts start",
+    "start": "PORT=8001 react-scripts start",
```

Comment:
So that if dashboard is running we can still launch the smart menu without port conflicts...